share/functions/psub.fish

```diff
@@ -54,12 +55,16 @@ function psub --description "Read from stdin into a file and output the filename
 		# that the command substitution exits without needing to wait for
 		# all the commands to exit
 		set dir (mktemp -d "$TMPDIR[1]"/.psub.XXXXXXXXXX); or return
-		set filename $dir/psub.fifo
+		set filename $dir/psub.fifo$suffix
```

Comment:
This $suffix needs to be quoted, otherwise the cartesian product kicks in and nixes the entire filename!